space/space_test.go

```diff
@@ -104,9 +104,21 @@ func (test *repoBBTest) TestDelete() {
 
 func (test *repoBBTest) TestList() {
 	_, orgCount, _ := test.list(nil, nil)
```

Comment:
what about including comments in the code to define the portions of the test:
````go
func (test *repoBBTest) TestList() {
  // given
  ...
  // when
  ...
  // then
  ...
}
````